### PR TITLE
Doc auto generated certificate will not work for diff protocols on diff interfaces

### PR DESCRIPTION
**Pull request by zhenlineo**
Document that if configuring different protocols on different interfaces that auto-generated certificates will not work

Document recommended workaround to use multi-host or wildcard certificate instead

### CODE REVIEW

File: community/server/src/docs/ops/security.asciidoc [hunk 1]
```diff
@@ -51,6 +51,10 @@ The Neo4j server includes built in support for SSL encrypted communication over
 The first time the server starts, it automatically generates a self-signed SSL certificate and a private key.
 Because the certificate is self signed, it is not safe to rely on for production use, instead, you should provide your own key and certificate for the server to use.
 
+[CAUTION]
+Using autogeneration of self-signed SSL certificates will not work if the Neo4j server has been configured with multiple connectors that do not bind to the same IP address.
+If you need to use multiple IP addresses, please configure certificates manually and consider using multi-host or wildcard certificates instead.
+
```
Comment (mariascharin) [inline community/server/src/docs/ops/security.asciidoc]: Small change in wording to increase clarity. This change needs to be done in the new manual repository. I have given you access. The file to modify in that repo is: src/main/operations-manual/security/security.asciidoc

Using auto-generation of self-signed SSL certificates will not work if the Neo4j server has been configured with multiple connectors that bind to different IP addresses. If you need to use multiple IP addresses, please configure certificates manually or consider using multi-host or wildcard certificates instead.

File: manual/contents/src/operations/configuration-reference.asciidoc [hunk 2]
```diff
@@ -16,6 +16,9 @@ Each connector has a unique key to identify it, denoted `(bolt-connector-key)` i
 
 include::{importdir}/neo4j-config-docs-docs-jar/ops/configuration-bolt-connector-attributes.asciidoc[]
 
+[CAUTION]
+Using autogeneration of self-signed SSL certificates will not work if the Neo4j server has been configured with multiple connectors that do not bind to the same IP address.
+If you need to use multiple IP addresses, please configure certificates manually and consider using multi-host or wildcard certificates instead.
```
Comment (mariascharin) [inline manual/contents/src/operations/configuration-reference.asciidoc]: This and the next change are in the right location. Pls just change the wording according to the above.

**Comment by boggle** [comment]
@mariascharin I've updated the wording but slightly differently as the original suggestion wasn't technically correct. Would you please also give me access to the new manual repository so that I can make the same change there? (also where is that repo?)

**Comment by mariascharin** [comment]
@boggle I have added you to the docs repo; you should receive an email.

**Comment by boggle** [comment]
@mariascharin thank you, I've pushed an update to that repo, too.

**Pull request merged**